ServerlessLibraryAPI/ClientApp/src/services/library.service.js

```diff
@@ -37,3 +35,17 @@ function submitNewSample(item) {
   };
   return fetch("/api/library", requestOptions).then(handleResponse);
 }
+
+function updateUserSentimentStats(id, likeChanges, dislikeChanges) {
+  const requestOptions = {
+    method: "PUT",
+    body: '"' + id + '"',
+    headers: {
+      "Content-Type": "application/json"
+    }
+  };
+  return fetch(
+    "/api/metrics/sentiment/" + likeChanges + "/" + dislikeChanges,
```

Comment:
IMHO the likes and dislikes should go into the body

Reply:
yup